pkg/proxy/metaproxier/meta_proxier.go

```diff
@@ -103,7 +103,7 @@ func (proxier *metaProxier) OnServiceSynced() {
 func (proxier *metaProxier) OnEndpointsAdd(endpoints *v1.Endpoints) {
 	ipFamily, err := endpointsIPFamily(endpoints)
 	if err != nil {
-		klog.Warningf("failed to add endpoints %s/%s with error %v", endpoints.ObjectMeta.Namespace, endpoints.ObjectMeta.Name, err)
+		klog.V(4).Infof("failed to add endpoints %s/%s with error %v", endpoints.ObjectMeta.Namespace, endpoints.ObjectMeta.Name, err)
```

Comment:
Maybe we should use V(1)?
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
> klog.V(1) - A reasonable default log level if you don't want verbosity.

> Information about config (listening on X, watching Y)
> Errors that repeat frequently that relate to conditions that can be corrected (pod detected as unhealthy)

Reply:
@aojea I'm fine with V(4). `endpointsIPFamily()` returns errors when (a) there are no subsets, (b) there are no subset addresses, (c) the first subset's ready IP is empty.

If we look at validation, then (a) is valid eg endpoints don't need subsets, (b) is valid because there may be not-ready addresses but no ready addresses, and (c) should have been rejected by validation due to `validateEndpointAddress()` and `validation.IsValidIP()`.

eg, none of these things that endpointsIPFamily() returns should be logged with a Warningf() because they are either valid configurations of endpoints/subsets, or are cases that shouldn't be hit becuase they would have been rejected by validation.